tools/build.js

```diff
@@ -8,6 +8,7 @@ import docs from '../docs/build';
 import dist from './dist/build';
 import { copy } from './fs-utils';
 import { setExecOptions } from './exec';
+import { exec } from './exec';
```

Comment:
I don't know why I didn't see this before, but these two lines can be condensed to `import { exec, setExecOptions } from './exec';`